FWCore/Framework/src/Schedule.cc

```diff
@@ -620,7 +644,38 @@ namespace edm {
     std::map<std::string, std::vector<std::pair<std::string, int> > > outputModulePathPositions;
     reduceParameterSet(proc_pset, tns.getEndPaths(), modulesInConfig, usedModuleLabels,
                        outputModulePathPositions);
-    processEDAliases(proc_pset, processConfiguration->processName(), preg);
+    const bool hasAliases = processEDAliases(proc_pset, processConfiguration->processName(), preg);
+
+    // At this point all BranchDescriptions are created. Mark now the
+    // ones of unscheduled workers to be on-demand.
+    // TODO: what to do for alias branches???
```

Comment:
What is the plan to resolve this TODO?

Reply:
Thanks for catching, the comment itself is a leftover. The alias branches are already dealt with below on lines 667-676.